src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Numerics
+{
+    public static class ArrayTensorExtensions
+    {
+        public static DenseTensor<T> ToTensor<T>(this T[] array, bool reverseStride = false)
```

Comment:
This was probably covered in the FxDC review, but why is this `ToTensor` and not `ToDenseTensor`?

Reply:
Because an array is already dense, so its most likely that folks will want to make it a dense tensor.  Sparse tensors are more common when data is too large to represent as a dense tensor: in which case you're not likely to have a (dense) array representation.